SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -22,14 +22,16 @@
   <!-- **************************************************************************** -->
   <!-- Using tasks -->
   <!-- **************************************************************************** -->
-  <PropertyGroup Condition=" $(SonarQubeTempPath) != '' ">
+  <PropertyGroup Condition=" $(SonarQubeTempPath) != '' AND $(SonarQubeBuildTasksAssemblyFile) == '' ">
     <!-- Assume that the tasks assembly is in the same location as this targets file 
```

Comment:
Minor clean-up: I consolidated the conditions in the PropertyGroup rather than splitting them across the PropertyGroup and the property assignment.